ansible/configs/ocp4-workshop/software.yml

```diff
@@ -56,6 +56,7 @@
           package:
             name:
               - golang
+              - python2-boto
```

Comment:
What is the motivation for adding python2-boto ?

Reply:
To use Route53 module in Ansible